tests/conftest.py

```diff
@@ -57,20 +150,22 @@ def clean_etcd():
     subprocess.run(["etcdctl", "del", '""', "--from-key=true"])
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
Normally these are just `@pytest.fixture`. Is there a reason it's being called here?

Reply:
No reason, I just got carried away with the ones receiving parameters :disappointed:

Reply:
@minrk, I pushed some changes! Hope I got it right this time